core/src/main/kotlin/no/strooped/TextureSizes.kt

```diff
@@ -22,4 +22,16 @@ object TextureSizes {
         // pinLabel on JoinGameScreen
         fun pinLabelWidth(): Float = Gdx.graphics.width * (115f / 1080f)
         fun pinLabelHeight(): Float = Gdx.graphics.height * (64f / 2040f)
+
+        // loadCircle on LobbyScreen
+        fun loadWidth(): Float = Gdx.graphics.width * (130f / 1080f) // 105
+        fun loadHeight(): Float = Gdx.graphics.height * (130f / 2040f)
+
+        // loadCircle on LobbyScreen
+        fun exitGameButtonWidth(): Float = Gdx.graphics.width * (376f / 1080f)
+        fun exitGameButtonHeight(): Float = Gdx.graphics.height * (100f / 2040f)
+
+        fun adjustedFontSize(font: Float): Float {
```

Comment:
I’m not sure what this function actually gives as output. Obviously a float, but why does it use window width and height, including 3120 to calculate some value? 

And could we potentially not just use the standard 14pt or 16pt as base, and then scale from that value?

Reply:
because when using standard numbers it varies a lot on different screens. This is a way of keeping it in the same size on every possible screen you look at. Similar to what I have done with the texture sizes. 3120 is 1080 + 2040 which is the screen I was testing it on. I can explain more of this in the call but basically it returns the value that is needed so the font on THAT particular screen is the size _font_ (the parameter).

Reply:
Changed name of function to be more readable